build.gradle

```diff
@@ -214,13 +214,13 @@ dependencies {
 
     // FIXME 0.6 doesn't support jsonb; 0.7 doesn't work on Windows
 //    runtime group: 'com.impossibl.pgjdbc-ng', name: 'pgjdbc-ng', version: '0.6'
-    runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.16'
+    runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.23'
```

Comment:
if this bump is not required for test container, could you remove please. We can do that in another PR. So that we can release this PR as test only changes.

Reply:
👍

Reply:
Fixed in 640c299